CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## [1.9.4] - 2019-07-30
```

Comment:
Please update the date to `2019-08-01` if you're planning on releasing these changes by EOD.